### PR TITLE
To Do list: Finished list structure.

### PR DESCRIPTION
**Pull request by Zafron047**
Hi there! This pull request adds a new project based on webpack, which implements a To Do List application. Here's a summary of the changes:

- Created index.html file as a template using HTML Webpack Plugin.

- Added an empty placeholder for the To Do List in index.html.

- Implemented index.js file with an array of tasks, each represented by an object.

- Wrote a function in index.js to dynamically create HTML list items for each task.

- Rendered the list of tasks on page load in the designated placeholder, ordering them based on the task index.

- Added style.css file to provide the necessary styling for the To Do List.

Please review and give me the opportunity to merge this pull request. 

Thank you!
Zafron. 😄

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,21 +1,62 @@
...
-    element.appendChild(myIcon);
-  
-    return element;
+// import _ from 'lodash';
+
+// let cloud = 'cloud';
+const array = JSON.parse(localStorage.getItem('Data')) || [];
+const listInput = document.querySelector('#listInput');
+let counter = 1;
+const list = document.querySelector('#list');
```
Comment (souad988) [inline src/index.js]: Good job on making all the requirments , only this counter should be initialized to 0  as here the first item gets 2 as index

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Zafron047 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good PR description
Setup webpack correctly
Use webpack css loader

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Using subfolders to separate styling files , images , js and html to make your work better organized .

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Zafron047) [reply src/index.js]: @souad988 Ohh my bad 😄 This mistake happened while fixing the eslint. I forgot to update in the method.
Thanks for your kind effort and direction. 😺

**Review by osamaashraf6 — APPROVED** [review]
Hi @Zafron047 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (3)

## Highlights🎉🎉

- [ ] Good setting up the project using webpack💯
- [ ] Use webpack css loader🚀🟠
- [ ] Descriptive pull request👍


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zafron047** [comment]
@souad988 Thanks. I will merge it now. 👍 😄

**Pull request merged**